uber/tests/import_test_data.py

```diff
@@ -3,6 +3,56 @@
 from hotel.models import *
 from guests.models import *
 
+from uber.config import c, create_namespace_uuid
```

Comment:
The changes in `import_test_data.py` are unrelated. I wanted to get the test data import working again so I could test how long things took with hundreds of groups.